content/docs/getting-started/installing.md

```diff
@@ -5,64 +5,85 @@ weight: 2
 
 ## From extension stores
 
-Scratch Addons is available in both Chrome Web Store and Add-ons for Firefox.
+Scratch Addons is available in both Chrome Web Store and Firefox Add-ons.
```

Comment:
The usage of AMO is already considered a while back, and Add-ons for Firefox is the chosen one.

Reply:
Fixed, will commit soon.